news/2019-11-11-project-loved-week-of-november-10th.md

```diff
@@ -0,0 +1,156 @@
+
+[![](/wiki/shared/news/2019-11-11-project-loved-week-of-november-10th/mania/2-gems-pack-14-ln-master-6th.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[Gems - Gems Pack 14 - LN Master 6th](https://osu.ppy.sh/beatmapsets/597080#mania) by [gemboyong](https://osu.ppy.sh/users/3518460)  
+168 BPM, 5:28 | [4K] 4.09★, [4K] 5.62★, [4K] 5.80★, [4K] 6.83★
+
+It's about time we bring in a pack from Onta_Metalslug (previously, and more well-known as gemboyong), and the choice we make for it is *Gem Pack 14*, or his 6th LN Master Pack. This pack contains four 7k inspired long note charts, ranging in difficulty from rather simple in Pink Gold, to brutally difficult and requiring great consistency in Other Side and Tsubomi. Because of the difficulty range and approachability of the charts in all of gem's LN Master packs, they have been extremely useful over the course of the past 2 years as LN training for aspiring high level LN players as well as all-rounders. The sheer popularity, recognizability, and usefulness of these packs has developed and been maintained over time, even with the influx of new LN maps and LN mappers. This combination of qualities more than makes Gem's LN Master 6th a worthy pack for inclusion in the Loved section.
+
+[![](/wiki/shared/news/2019-11-11-project-loved-week-of-november-10th/mania/3-shinkai-shoujou.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[Miku (Vocal by Nobunaga) - Shinkai Shoujou](https://osu.ppy.sh/beatmapsets/149086#mania) by [Cuppp](https://osu.ppy.sh/users/2345293)  
+137 BPM, 3:31 | [7K] 5.31★
+
+Prepare yourselves for yet another serving of Cuppp noodles cause we're having *Shinkai Shoujou* for this round! While being a much easier map compared to most of their other stuff, *Shinkai Shoujou* has quite the exposure from seeing play in MWC 7K 2017 semis and an early iteration of the [Jinjin dans](https://osu.ppy.sh/users/3360737). At a seemingly harmless speed of 137 BPM, the map requires some O2Jam fundamentals when it comes to finger independence. Column specific inversions at 1|2|3, 5|6|7, and 1|3|5|7 are used noticeably with occasional anchors that gets paired with 1/4 trilling. Not very scary but it can be a little tasking while trying to hold other noodles. Topped off with some isolated quirky patterns, *Shinkai Shoujou* may be a neat little chart to look back at for those who struggled to master LNs back in the day.
```

Comment:
```suggestion
Prepare yourselves for yet another serving of Cuppp noodles cause we're having *Shinkai Shoujou* for this round! While being a much easier map compared to most of their other stuff, *Shinkai Shoujou* has quite the exposure from seeing play in MWC 7K 2017 semis and an early iteration of the [Jinjin](https://osu.ppy.sh/users/3360737) dans. At a seemingly harmless speed of 137 BPM, the map requires some O2Jam fundamentals when it comes to finger independence. Column specific inversions at 1|2|3, 5|6|7, and 1|3|5|7 are used noticeably with occasional anchors that gets paired with 1/4 trilling. Not very scary but it can be a little tasking while trying to hold other noodles. Topped off with some isolated quirky patterns, *Shinkai Shoujou* may be a neat little chart to look back at for those who struggled to master LNs back in the day.
```